### ISSUE TITLE
Just query requires write access to redis

### ISSUE THREAD

**Issue by jahlives** [pagekite/PyPdnsRedis#4]
I use pdns_redis.py on my server which runs a redis slave instance. If I just query pdns for a record I get SERVFAIL and in the logs

> Dec  9 23:21:18 rbl3 pdns_server: Coprocess: Internal Error: You can't write against a read only slave.
> Dec  9 23:21:18 rbl3 pdns_server: [PIPEBackend] coprocess returned a FAIL
> Dec  9 23:21:18 rbl3 pdns_server: Backend reported condition which prevented lookup (coprocess returned a FAIL) sending out servfail

I just wonder if its really necessary to write to redis in case of answering a query? 
I tried the ```-R``` and ```-W``` parameters but the error persists. So for me it seems that ```-W``` does not have any effect. I pointed it to the master redis but still got the error upon querying.

**Comment by pagekite** [comment]
Fair point. Making this configurable would make sense! I would accept a PR for that.
